drivers/hmis/app/models/hmis/ce/match/client_field_map.rb

```diff
@@ -35,7 +35,36 @@ def all
             GrdaWarehouse::Hud::Client.days_homeless(client_id: c.id)
           end,
         },
-        # Add support for more client fields here...
+        # Array of Project Types at which the Client has an open Enrollment, including WIP enrollments.
+        open_enrollment_project_types: {
+          instance_value: ->(c) do
+            Hmis::Hud::Enrollment.joins(client: :warehouse_client_source).
+              where(warehouse_clients: { destination_id: c.id }).
+              open_including_wip.
+              joins(:project).
```

Comment:
simple performance optimization to select distinct project types (a client might have many enrollments). Would also apply to the other project type queries
```suggestion
              joins(:project).
              distinct.
```